recipe/meta.yaml

```diff
@@ -43,6 +43,7 @@ outputs:
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
       host:
         - cuda-version {{ cuda_version }}
+        - expat                                 # [aarch64]
```

Comment:
Saw this error [on CI]( https://app.travis-ci.com/github/conda-forge/cuda-gdb-feedstock/builds/264002247#L793 )

```
  ERROR (cuda-gdb,bin/cuda-gdb): $RPATH/libexpat.so.1 not found in packages, sysroot(s) nor the missing_dso_whitelist.
```

<br>

This only happens on Linux `aarch64`. So have added `expat` for those builds only